content/src/licenses/cosmetology.md

```diff
@@ -17,6 +16,12 @@ callToActionText: Renew Shop Registration
 - Current business address
 - Pay the associated fees for the registration renewal
 
-> **Once completed, you will have:**
->
-> - A renewed cosmetology shop registration valid for two years. It will be mailed to your business mailing address on file.
+:::callout{ icon="false" calloutType="conditional" }
+
+- A renewed cosmetology shop registration valid for two years. It will be mailed to your business mailing address on file.
+
+:::
+
+:::callout{ header="What if I don't renew my registration?" icon="false" calloutType="conditional" }
+If you don't renew your registration 1 month after the expiration, your license will be suspended and you won't be able to legally run your business. You need to contact your licensing board for reinstatement procedures and associated fees.
+:::
```

Comment:
```suggestion
:::callout{ icon="false" calloutType="conditional" }

- A renewed cosmetology shop registration valid for two years. It will be mailed to your business mailing address on file.

:::
```